lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -277,6 +279,18 @@ define([
         ));
     };
 
+    createMockPrivateTopic = function(options) {
+        return _.extend(
+            {
+                id: testPrivateTopicID,
+                name: 'Test Private Topic 1',
+                description: 'Test private topic description 1',
+                type: 'private_managed'
+            },
+            options
+        );
+    };
```

Comment:
I worry that this and the above mock creator (I grant that this is accurately following the existing workflow), both are very specific, and would not work at all in the case that someone needed to test a scenario with Multiple private_managed or instructor_managed topics...  Not sure this is a change request for this PR, so much as a note that we should think about making these more generalized.

Reply:
I agree. I think they were both too rigidly defined and don't really provide any extra functionality. I removed them in favor of overriding the existing `createMockTopic` helper which can be extended to create different topic types and chained to create multiple topics of different types.